### PR TITLE
adds subtler emote that wraps around adjacent tables

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

also removes audio/visual toggle for subtler because it's omni anyways

## Why It's Good For The Game

someone asked me a long time ago and i was reminded

## Changelog
:cl: ERP mains
add: Subtler Around Table is now a verb
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
VERB BLOAT

File: code/modules/mob/say_vr.dm [hunk 1]
```diff
@@ -111,6 +102,77 @@ proc/get_top_level_mob(var/mob/S)
...
+			processing |= other
+		other = locate() in get_step(T, SOUTH)
+		if(other)
+			processing |= other
+		other = locate() in get_step(T, WEST)
+		if(other)
+			processing |= other
+		other = locate() in get_step(T, EAST)
+		if(other)
+			processing |= other
```
Comment (Putnam3145) [inline code/modules/mob/say_vr.dm:173]: so uh is processing supposed to be used or

**Comment by tiramisuapimancer** [comment]
what precisely the fuck does subtler around table do it is not clear to me but i may be stupid

**Review by DeltaFire15 — APPROVED** [review]
Fine kev can have his dare dice subtler.

**Pull request merged**